### DISCUSSION TITLE
Out of memory when batch calculating MRT using SOLWEIG

### DISCUSSION THREAD

**Discussion by qinqinqinj** [UMEP-dev/UMEP#615] [Q&A]
I am using QGIS version 3.34,I am running out of memory after a dozen or so calculations during batch processing, is there any solution?
![图片1](https://github.com/UMEP-dev/UMEP/assets/147718158/2b651e20-6bbb-468f-aa70-0b9bf66fbd7d)

**Comment by biglimp** [comment]
I had a student who ran SOLWEIG in batch without any issues a couple of years ago. He ran a couple of 100 domains of 700x700 gridcells. Maybe he also used an earlier version using a isotrophic sky which requires less memory. Dont remember. His thesis is found here (https://gupea.ub.gu.se/handle/2077/74598).

**Comment by qinqinqinj** [comment]
Thanks for the reply, it may take more attempts to find out what the problem is
